pkg/storaged/client.js

```diff
@@ -769,6 +790,94 @@
 
     client.vdo_overlay = vdo_overlay();
 
+    /* HACK - clevis */
+
+    function clevis_overlay() {
+        var self = {
+            start: start,
+
+            info: { },
+
+            find_by_block: find_by_block,
+
+            remove: remove,
+            get_adv: get_adv,
+            add: add,
+            check_key: check_key,
+            replace: replace,
+            unlock: unlock
+        };
+
+        function spawn_tool(args) {
+            return cockpit.spawn([ "python3", "--", "-" ].concat(args), { superuser: "try", err: "message" })
```

Comment:
Deliberately not using `python.spawn()` here? Any chance that this will get backported to RHEL 7? Or is this just a temporary shim until clevis/tang get more API (the comment in the py helper suggest that)?

Reply:
It's all temporary.  The plan is to add enough API to UDisks2 for this.